src/main/java/org/gridsuite/modification/server/service/FilterService.java

```diff
@@ -65,6 +70,15 @@ public List<FilterEquipments> exportFilters(List<UUID> filtersUuids, UUID networ
         }
     }
 
+    public Map<UUID, FilterEquipments> getUuidFilterEquipmentsMap(Network network, Map<UUID, String> filters) {
+        String workingVariantId = network.getVariantManager().getWorkingVariantId();
+        UUID uuid = ((NetworkImpl) network).getUuid();
+        return exportFilters(new ArrayList<>(filters.keySet()), uuid, workingVariantId)
+                .stream()
+                .peek(t -> t.setFilterName(filters.get(t.getFilterId())))
```

Comment:
Is peek appropriate ? Shouldn't it only be used for debugging ?

Reply:
As mentioned here: https://www.baeldung.com/java-streams-peek-api, i think the use of peek is discouraged when there is no terminal operation. In this case we have one. I used it here because we want to change the inner state of element without changing the returned element itself.